2021-04-08 22:33:46,000 INFO    MainThread:8616 [wandb_setup.py:_flush():69] setting env: {}
2021-04-08 22:33:46,000 INFO    MainThread:8616 [wandb_setup.py:_flush():69] setting login settings: {}
2021-04-08 22:33:46,001 INFO    MainThread:8616 [wandb_init.py:_log_setup():336] Logging user logs to C:\Users\aleja\Desktop\Tareas\Reconocimiento Visual con Deep Learning\Tarea1\Git\cc7221\tarea1\src\wandb\run-20210408_223345-13rlbxai\logs\debug.log
2021-04-08 22:33:46,001 INFO    MainThread:8616 [wandb_init.py:_log_setup():337] Logging internal logs to C:\Users\aleja\Desktop\Tareas\Reconocimiento Visual con Deep Learning\Tarea1\Git\cc7221\tarea1\src\wandb\run-20210408_223345-13rlbxai\logs\debug-internal.log
2021-04-08 22:33:46,002 INFO    MainThread:8616 [wandb_init.py:init():369] calling init triggers
2021-04-08 22:33:46,002 INFO    MainThread:8616 [wandb_init.py:init():374] wandb.init called with sweep_config: {}
config: {}
2021-04-08 22:33:46,003 INFO    MainThread:8616 [wandb_init.py:init():418] starting backend
2021-04-08 22:33:46,003 INFO    MainThread:8616 [backend.py:_multiprocessing_setup():69] multiprocessing start_methods=spawn, using: spawn
2021-04-08 22:33:46,011 INFO    MainThread:8616 [backend.py:ensure_launched():123] starting backend process...
2021-04-08 22:33:46,088 INFO    MainThread:8616 [backend.py:ensure_launched():127] started backend process with pid: 21096
2021-04-08 22:33:46,089 INFO    MainThread:8616 [wandb_init.py:init():423] backend started and connected
2021-04-08 22:33:46,090 INFO    MainThread:8616 [wandb_init.py:init():463] updated telemetry
2021-04-08 22:33:46,091 INFO    MainThread:8616 [wandb_init.py:init():482] communicating current version
2021-04-08 22:33:47,161 INFO    MainThread:8616 [wandb_init.py:init():487] got version response 
2021-04-08 22:33:47,161 INFO    MainThread:8616 [wandb_init.py:init():495] communicating run to backend with 30 second timeout
2021-04-08 22:33:47,572 INFO    MainThread:8616 [wandb_init.py:init():520] starting run threads in backend
2021-04-08 22:33:47,704 INFO    MainThread:8616 [wandb_run.py:_console_start():1524] atexit reg
2021-04-08 22:33:47,705 INFO    MainThread:8616 [wandb_run.py:_redirect():1371] redirect: SettingsConsole.WRAP
2021-04-08 22:33:47,706 INFO    MainThread:8616 [wandb_run.py:_redirect():1414] Wrapping output streams.
2021-04-08 22:33:47,706 INFO    MainThread:8616 [wandb_run.py:_redirect():1438] Redirects installed.
2021-04-08 22:33:47,706 INFO    MainThread:8616 [wandb_init.py:init():544] run started, returning control to user process
2021-04-08 22:33:47,706 INFO    MainThread:8616 [wandb_config.py:__setitem__():141] config set learning_rate = 0.01 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x0000020287890A30>>
2021-04-08 22:33:47,706 INFO    MainThread:8616 [wandb_run.py:_config_callback():749] config_cb learning_rate 0.01 None
2021-04-08 22:33:52,579 INFO    MainThread:8616 [wandb_run.py:_atexit_cleanup():1494] got exitcode: 1
2021-04-08 22:33:52,580 INFO    MainThread:8616 [wandb_run.py:_restore():1466] restore
2021-04-08 22:33:52,826 INFO    MainThread:8616 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 938
  total_bytes: 938
}

2021-04-08 22:33:53,625 INFO    MainThread:8616 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 938
  total_bytes: 2084
}

2021-04-08 22:33:54,042 INFO    MainThread:8616 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 2086
  total_bytes: 2086
}

2021-04-08 22:33:54,148 INFO    MainThread:8616 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 2086
  total_bytes: 2086
}

2021-04-08 22:33:54,257 INFO    MainThread:8616 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 2086
  total_bytes: 2086
}

2021-04-08 22:33:54,366 INFO    MainThread:8616 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 2086
  total_bytes: 2086
}

2021-04-08 22:33:54,475 INFO    MainThread:8616 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 2086
  total_bytes: 2086
}

2021-04-08 22:33:54,587 INFO    MainThread:8616 [wandb_run.py:_wait_for_finish():1616] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 2086
  total_bytes: 2086
}

2021-04-08 22:33:55,683 INFO    MainThread:8616 [wandb_run.py:_show_files():1838] logging synced files
